docker/Dockerfile

```diff
@@ -15,6 +15,7 @@ ENV PATH="${PATH}:/opt/cargo/bin"
 ENV RUST_TOOLCHAIN="stable"
 RUN rustup toolchain install "${RUST_TOOLCHAIN}"
 RUN rustup default "${RUST_TOOLCHAIN}"
+RUN cargo install cargo-deb || true
```

Comment:
`|| true` lets us continue if `cargo-deb` is already present due to e.g. caching.